modules/oe_translation_poetry/modules/oe_translation_poetry_mock/oe_translation_poetry_mock.routing.yml

```diff
@@ -0,0 +1,20 @@
+oe_translation_poetry_mock.server:
+  path: '/poetry-mock/server'
+  defaults:
+    _controller: 'Drupal\oe_translation_poetry_mock\Controller\PoetryMockController::server'
+  requirements:
+    _permission: 'access content'
+
+oe_translation_poetry_mock.server_types:
```

Comment:
What is the purpose of this route?

Reply:
Unclear at the moment, it exists in D7. There is a todo to remove it.